manifests/i/IntelCorporation/IntelSystemSupportUtility/2.5.0.15/IntelCorporation.IntelSystemSupportUtility.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: IntelCorporation.IntelSystemSupportUtility
+PackageVersion: 2.5.0.15
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: https://downloadmirror.intel.com/25293/eng/ssu.exe
+  InstallerSha256: 71306A306996004E13569CA0D6D5FBF88AC1D740227FDD899F8D1A9F81B99C6E

```

Incase it dosn't support x86 we need to split the architecture
```suggestion
- Architecture: x64
  InstallerType: nullsoft
  InstallerUrl: https://downloadmirror.intel.com/25293/eng/ssu.exe
  InstallerSha256: 71306A306996004E13569CA0D6D5FBF88AC1D740227FDD899F8D1A9F81B99C6E
- Architecture: x86
  InstallerType: nullsoft
  InstallerUrl: https://downloadmirror.intel.com/25293/eng/ssu.exe
  InstallerSha256: 71306A306996004E13569CA0D6D5FBF88AC1D740227FDD899F8D1A9F81B99C6E
```